crates/bevy_color/src/linear_rgba.rs

```diff
@@ -319,6 +319,7 @@ impl encase::ShaderType for LinearRgba {
         encase::private::Metadata {
             alignment,
             has_uniform_min_alignment: false,
+            is_pod: false,
```

Comment:
Isn't this PoD?

Reply:
Removing from merge queue until this is resolved. It looks like it should be pod.

Reply:
```suggestion
            is_pod: true,
```